CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Station Simulator Releases #
 
+### Version 0.4.1
+
+- Upgraded simulator to Java 11
```

Comment:
this does not sounds like a minor change

Reply:
Yes my bad, but when i realised the jar was already published on bintray. I don't think it would make sense now to publish a new one with version 0.5